src/plone/restapi/tests/test_dxcontent_serializer.py

```diff
@@ -498,6 +499,26 @@ def test_allow_discussion_obj_instance_allows_global_enabled(self):
         self.assertIn("allow_discussion", obj)
         self.assertEqual(True, obj["allow_discussion"])
 
+    def isPlone61OrAbove(self):
+        """Check if the Plone version is 6.1 or above."""
+        installer = get_installer(self.portal)
+        return installer.getVersion() >= "6.1"
```

Comment:
You can create a constant similar to this:

https://github.com/plone/plone.restapi/blob/f90329dfe49aa483517861a610b5175d68e0b6be/src/plone/restapi/deserializer/site.py#L15-L17

Reply:
do I need to create the marker for 6.1 or it is already created?